content/utilities/mark-friends/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Mark Friends
```

Comment:
How about hilight friends? All the other plugins use the word highlight and it might be more searchable?